guidelines/_sources/04_report-prompts.tex

```diff
@@ -4,8 +4,11 @@
 
 \subsection{Report Prompts and their Development}
 
+\comment{I found it a bit hard to understand when to use \textbf{MUST} and \textbf{SHOULD} in this section. I think we need to clarify this. Sorry, computer networks lecturer's taking over, but we could follow the RFC2119 that defines requirements levels in Internet protocol specifications (https://datatracker.ietf.org/doc/html/rfc2119). Something along this line?  \textbf{MUST}: An absolute requirement for reproducibility.  \textbf{SHOULD}: A strongly recommended practice that may be omitted with justification. \textbf{MAY}: An optional practice.}
+
 \subsubsection{Recommendations}
-Prompts are critical in empirical software engineering studies involving LLMs. Depending on the task, prompts may include various types of content, such as source code, execution traces, error messages, natural language descriptions, or even screenshots and other multi-modal inputs. These elements significantly influence the model’s output, and understanding how exactly they were formatted and integrated is essential for transparency and reproducibility.
+\comment{These examples are fine but as scenarios they might be more representative of developers rather than researchers' activities? What if we took of prompts that are relevant for the examples from the Section Study Types (annotators, raters, synthesis, etc.)?}
```

Comment:
Thanks @bateman! Which examples were you referring to specifically?

Reply:
@ctreude I gave it another read and this time i didn't get the same impression. The text already mentions "natural language descriptions, or even screenshots and other multi-modal inputs." I think it is already looking good and we can safely ignore my comment. Sorry about that.